atom/browser/api/atom_api_session.cc

```diff
@@ -106,6 +140,7 @@ mate::ObjectTemplateBuilder Session::GetObjectTemplateBuilder(
     v8::Isolate* isolate) {
   return mate::ObjectTemplateBuilder(isolate)
       .SetMethod("resolveProxy", &Session::ResolveProxy)
+      .SetMethod("clearCache", &Session::ClearCache)
```

Comment:
If the final API name is `clearStorageData`, we should use `clearStorageData` here too.